bootstrap_form.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.description = "bootstrap_form is a rails form builder that makes it super "\
                   "easy to create beautiful-looking forms using Bootstrap 5"
   s.license     = "MIT"
+  s.metadata    = { "rubygems_mfa_required" => "true" }
```

Comment:
Oh yeah, I probably should have mentioned this. 😬 It's something that Rubocop wanted us to add. Maybe, if we can't do this, we can put a "disable warning" block here. But on the other hand, I think the goal here is to make the rubygems login more secure to prevent hacks.

Reply:
I'm all for better security. When I finally pushed the 5.0 version of the gem I was wondering if I was going to have to give 2FA, but it just worked so I didn't question it. Glad you followed up.